src/inc/corinfo.h

```diff
@@ -213,11 +213,11 @@ TODO: Talk about initializing strutures before use
     #define SELECTANY extern __declspec(selectany)
 #endif
 
-SELECTANY const GUID JITEEVersionIdentifier = { /* e5708e9e-dd18-4287-8745-5d10ff2697cf */
-    0xe5708e9e,
-    0xdd18,
-    0x4287,
-    { 0x87, 0x45, 0x5d, 0x10, 0xff, 0x26, 0x97, 0xcf }
+SELECTANY const GUID JITEEVersionIdentifier = { /* 1348cf5b-d47f-4102-87a7-68020d32d677 */
```

Comment:
> So, this is not considered to be a change in the JIT/EE interface. Is it so

The guid only needs to be changed if new JIT does not work with old runtime; or vice versa.